benchmark/benchmark-tidb-using-sysbench.md

```diff
@@ -130,7 +132,7 @@ sysbench --config-file=config oltp_point_select --tables=32 --table-size=1000000
 
 To warm data, we load data from disk into the block cache of memory. The warmed data has significantly improved the overall performance of the system. It is recommended to warm data once after restarting the cluster.
 
-Sysbench 1.0.14 does not provide data warming, so it must be done manually. If you are using [Sysbench of the master version](https://github.com/akopytov/sysbench/tree/master), you can use the data warming feature included in the tool itself.
+Sysbench 1.0.20 does not provide data warming, so it must be done manually. If you are using [Sysbench of the master version](https://github.com/akopytov/sysbench/tree/master), you can use the data warming feature included in the tool itself.
```

Comment:
@dbsid Looks like for 1.0.20 and 1.0.14 this isn't correct.

```
$ sysbench oltp_read_only 
sysbench 1.1.0-df89d34 (using bundled LuaJIT 2.1.0-beta3)

FATAL: ./src/lua/oltp_common.lua:28: Command is required. Supported commands: prepare, warmup, run, cleanup, help
```

Looks like `sysbench oltp_read_only warmup ...` works fine.

Should we change the text here?

Reply:
Yes, I agree to change the text. Users can also use the built-in warmup command.

Reply:
Done in fe7bdf8e7e7bce43f78abfa3fff2e7ef641c4fea